lib/mongo/crypt/binding.rb

```diff
@@ -87,6 +94,11 @@ class Binding
       #   @return [ String ] A version string for libmongocrypt.
       attach_function :mongocrypt_version, [:pointer], :string
 
+      if Gem::Version.new(mongocrypt_version(nil)) < MIN_LIBMONGOCRYPT_VERSION
```

Comment:
Can this be something like

```
if (actual_version = Gem::Version.new(mongocrypt_version(nil))) < MIN_LIBMONGOCRYPT_VERSION

```

to avoid copy-pasting the entire expression in the exception message below please.

Reply:
Done